test/main/vct/test/integration/examples/SYCLSpec.scala

```diff
@@ -4,14 +4,32 @@ import vct.test.integration.helper.VercorsSpec
 
 class SYCLSpec extends VercorsSpec {
   vercors should verify using silicon example "concepts/sycl/kernels/BasicKernel.cpp"
-  vercors should error withCode "multipleKernels" example "concepts/sycl/kernels/MultipleKernelsInCommandGroup.cpp"
+  vercors should error withCode "syclNoMultipleKernels" example "concepts/sycl/kernels/MultipleKernelsInCommandGroup.cpp"
   vercors should verify using silicon example "concepts/sycl/kernels/NDRangeKernel.cpp"
-  vercors should error withCode "kernelForkPre" example "concepts/sycl/kernels/NegativeRange.cpp"
-  vercors should error withCode "missingKernel" example "concepts/sycl/kernels/NoKernelInCommandGroup.cpp"
-  vercors should error withCode "kernelForkPre" example "concepts/sycl/kernels/NonDivisibleNDRange.cpp"
-  vercors should error withCode "incorrectParallelForLambdaArgument" example "concepts/sycl/kernels/NonMatchingRangeItem.cpp"
-  vercors should error withCode "incorrectParallelForLambdaArgument" example "concepts/sycl/kernels/NonMatchingRangeItemDimensions.cpp"
-  vercors should error withCode "nonSYCLCodeInCommandGroup" example "concepts/sycl/kernels/NonSYCLCodeInCommandGroup.cpp"
+  vercors should error withCode "syclKernelRangeInvalid" example "concepts/sycl/kernels/NegativeRange.cpp"
+  vercors should error withCode "syclMissingKernel" example "concepts/sycl/kernels/NoKernelInCommandGroup.cpp"
+  vercors should error withCode "syclKernelRangeInvalid" example "concepts/sycl/kernels/NonDivisibleNDRange.cpp"
+  vercors should error withCode "syclIncorrectParallelForLambdaArgument" example "concepts/sycl/kernels/NonMatchingRangeItem.cpp"
+  vercors should error withCode "syclIncorrectParallelForLambdaArgument" example "concepts/sycl/kernels/NonMatchingRangeItemDimensions.cpp"
+  vercors should error withCode "syclNoExtraCodeInCommandGroup" example "concepts/sycl/kernels/NonSYCLCodeInCommandGroup.cpp"
   vercors should fail withCode "syclItemMethodPreFailed" using silicon example "concepts/sycl/kernels/TooHighKernelDimension.cpp"
-  vercors should error withCode "kernelForkPre" example "concepts/sycl/kernels/ZeroNDRange.cpp"
+  vercors should error withCode "syclKernelRangeInvalid" example "concepts/sycl/kernels/ZeroNDRange.cpp"
+
+  vercors should error withCode "syclAccessorInsufficientReferencePermission" example "concepts/sycl/accessors/AccessorGetRangeDimensionOutOfBounds.cpp"
+//  vercors should verify using silicon example "concepts/sycl/accessors/AllAccessModes.cpp"
```

Comment:
I commented out these two tests because they require the VerCors `--no-infer-heap-context-into-frame` flag to verify without VerCors throwing an error. And I do not know of a way to specify in the test which VerCors flags should be enabled.

Reply:
We should probably think about adding support for that, but can you also report the error as an issue please?